namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-estate-preprod/00-namespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: estate-preprod
```

Comment:
Can you please change it to prison-estate-preprod, same as your folder name.
OR if you like to keep this name, can you please change your folder name.